easybuild/easyconfigs/t/triqs_cthyb/triqs_cthyb-2.2.0-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,70 @@
+easyblock = 'CMakeMake'
+
+name = 'triqs_cthyb'
```

Comment:
Why not just `cthyb`?

Reply:
We made this choice for the package names in our [debian package repository](https://users.flatironinstitute.org/~ccq/triqs/). It would be good to match the naming convention to avoid confusion. We were worried about nameclashes with other codes which is why we avoided the shorter name.

Reply:
I propose to switch to the new naming scheme also for the easyconfig files and to remove the previous TRIQS eb. I think this is the best course of action in the long run.

Reply:
The policy in EasyBuild is to use the official software name however, which is also what we did initially for TRIQS. I would strongly prefer stricking to that (and not removing existing easyconfigs).

Certainly for `TRIQS` itself we should keep using `"TRIQS"` (see https://triqs.github.io/triqs/latest/, https://github.com/TRIQS/triqs, ...).
For the others, I could live with `TRIQS-cthyb` if you strongly prefer keeping them "together".

Note that the "user-facing" name of the installations is under the control of whoever is configuring EasyBuild, since a custom module naming scheme can be used (which uses lowercase names, uppercase, original, you name it).

Reply:
I have renamed the easyconfigs and module names to the TRIQS_* scheme.